### PR TITLE
added flow logic for business structure cluster

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: app/uk/gov/hmrc/offpayroll/models/BusinessStructureCluster.scala [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.offpayroll.models
+
+/**
+  * Created by peter on 15/12/2016.
```
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/models/BusinessStructureCluster.scala]: Copy and paste of my name and creation date in all files I think. No big deal but better to tidy this up

File: app/uk/gov/hmrc/offpayroll/models/BusinessStructureCluster.scala [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  */
+object BusinessStructureCluster extends Cluster {
+
+  /**
+    * Use this value to informatively name the cluster and use as a key to tags
+    */
+  override def name: String = "businessStructure"
+
+  override def clusterID: Int = 0
```
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/models/BusinessStructureCluster.scala]: this should be the 5th cluster zero indexed

File: app/uk/gov/hmrc/offpayroll/models/BusinessStructureCluster.scala (hunk 1 above)
Comment (habeeb01) [inline app/uk/gov/hmrc/offpayroll/models/BusinessStructureCluster.scala]: comment removed

File: app/uk/gov/hmrc/offpayroll/models/BusinessStructureCluster.scala (hunk 2 above)
Comment (habeeb01) [inline app/uk/gov/hmrc/offpayroll/models/BusinessStructureCluster.scala]: I have updated this. clusterId is now set to 5 for businessStructure.

Comment (habeeb01) [inline app/uk/gov/hmrc/offpayroll/models/BusinessStructureCluster.scala]: PartAndParcel clusterId is now set to 4

**Pull request merged**